File: ManagerFrontApp/src/app/data/floor.ts
import { Room } from "./room";

export interface Floor{
    level: string,
    info?: string,
    rooms: Room[],
    labels?: FloorLabel[],
    highlightedRoomId?: number
}


export interface FloorLabel{
    x: number,
    y: number,
    text: string
}

File: WebAppCore/Hospital/GraphicalEditor/Model/Floor.cs
using Hospital.RoomsAndEquipment.Model;
using System;
using System.Collections.Generic;
using System.Text;

namespace Hospital.GraphicalEditor.Model
{
    public class Floor
    {
        public int Id { get; set; }
        public string Level { get; set; }
        public string Info { get; set; }
        public int BuildingId { get; set; }

    }
}
